reactome_gnn/network.py

```diff
@@ -1,28 +1,44 @@
 import os
 import json
+import urllib.request
 from collections import defaultdict, deque, namedtuple
 
 import networkx as nx
 import matplotlib.pyplot as plt
+from reactome2py import fiviz, analysis
 
 
 class Network:
     
     Info = namedtuple('Info', ['name', 'species', 'type', 'diagram'])
 
-    def __init__(self, txt_path='data/ReactomePathwaysRelation.txt', json_path='data/9606'):
-        self.graph_txt = self.parse_txt(txt_path)
-        self.graph_dict, self.pathway_info, self.parent_dict = self.parse_json(json_path)
+    def __init__(self, txt_url='https://reactome.org/download/current/ReactomePathwaysRelation.txt',
+                 json_url='https://reactome.org/ContentService/data/eventsHierarchy/9606'):
+        self.graph_txt = self.parse_txt(txt_url)
+        self.graph_dict, self.pathway_info, self.parent_dict = self.parse_json(json_url)
         self.name_to_id = self.get_name_to_id()
+        self.markers = 'RAS,MAP,IL10,EGF,EGFR,STAT'
```

Comment:
gene variants or study gene marker(s) are always user defined. You _may_ want the user to provide markers if they don't exist.

Reply:
Alright, so while performing enrichment analysis, I can add code so that user inputs which markers they want, instead of hardcoding them. This I took from:
https://github.com/reactome/reactome2py/blob/master/demo/microbiologist-clinician.ipynb